crates/typst/src/layout/layout.rs

```diff
@@ -43,8 +43,11 @@ use crate::syntax::Span;
 /// })
 /// ```
 ///
-/// Note that this function will provide an infinite width or height if one of
-/// the page width or height is `auto`, respectively.
+/// Note that the provided width or height will be infinite in case the
+/// corresponding page dimension is set to `{auto}`.
+///
+/// Moreover, this function also provides [context]($context) to its argument,
+/// so you don't need to use it in combination with the `context` keyword.
```

Comment:
Not 100% sure on this one. I feel like there could be some better way to describe this behavior.

An example (e.g. using `here()`) could be helpful.

```suggestion
/// Moreover, `layout` also provides [context]($context) to its argument's
/// function, meaning you don't need to use it in combination with the
/// `context` keyword.
```

Reply:
`[context]($context)` can just be `[context]` if I'm not mistaken

Reply:
The wordings seem very similar, I think either is fine.